Description: Disable tests for now
 Tests fail when building the package, but run fine otherwise.
 Will revisit.
 .
 hsluv (0.0.2-2) yakkety; urgency=low
 .
   * source package automatically created by stdeb 0.8.5
Author: Steve Kondik <shade@chemlab.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-02-13

--- hsluv-0.0.2.orig/setup.py
+++ hsluv-0.0.2/setup.py
@@ -20,6 +20,6 @@ setup(
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 3"
     ],
-    py_modules=["hsluv"],
-    test_suite="tests.hsluv_test"
+    py_modules=["hsluv"]
+    #test_suite="tests.hsluv_test"
 )
